### PR TITLE
new records

### PR DESCRIPTION
**Pull request by thea-m**

### CODE REVIEW

File: new/INS0981MDDL.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        <revisionDesc>
+            <change who="DR" when="2022-09-22">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="privateHouse" subtype="institution">
+                    <placeName>Private collection of Mamhǝr Damǝss (Dabra Libānos)</placeName>
+                    <country ref="wd:Q115">Ethiopia</country>
```
Comment (eu-genia) [inline new/INS0981MDDL.xml:49]: could you also add the administrative regions? here and elsewhere?

File: new/INS0982MAML.xml [hunk 2]
```diff
@@ -0,0 +1,54 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="INS0982MAML" xml:lang="en" type="ins">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Private collection of Mamhǝr ʾAʾmǝro (Marṭul Māryām)</title>
```
Comment (eu-genia) [inline new/INS0982MAML.xml:8]: Marṭula

File: new/INS0982MAML.xml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        <revisionDesc>
+            <change who="DR" when="2022-09-22">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="privateHouse" subtype="institution">
+                    <placeName>Private collection of Mamhǝr ʾAʾmǝro (Marṭula Māryām)</placeName>
+                    <country ref="wd:Q115">Ethiopia</country>
```
Comment (eu-genia) [inline new/INS0982MAML.xml:49]: region + locality LOC2958Ennaba + Marṭula Maryam (LOC4594Martul)

File: new/INS0981MDDL.xml [hunk 4]
```diff
@@ -0,0 +1,55 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="INS0981MDDL" xml:lang="en" type="ins">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Private collection of Mamhǝr Damǝss (Dabra Libānos)</title>
```
Comment (eu-genia) [inline new/INS0981MDDL.xml:8]: I guess the private owners also need a person record, with a relation to INS and to MSS as role `owner`

File: new/INS0983MHDL.xml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="DR" when="2022-09-22">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="privateHouse" subtype="institution">
+                    <placeName>Private collection of Hayla Māryām (Dabra Libānos)</placeName>
```
Comment (eu-genia) [inline new/INS0983MHDL.xml:48]: Ḫayla?

File: new/INS0983MHDL.xml [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+                    <country ref="wd:Q115">Ethiopia</country>
+                    <settlement ref="LOC2297DabraL">Dabra Libānos</settlement>
```
Comment (eu-genia) [inline new/INS0983MHDL.xml:50]: see above

File: new/INS0984MLQM.xml [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="church" subtype="institution">
+                    <placeName xml:lang="gez" xml:id="n1">መንበረ፡ ልዑል፡ ቅዱስ፡ ማርቆስ፡</placeName>
+                    <placeName xml:lang="gez" corresp="#n1" type="normalized">Manbara Lǝʿul Qəddus Mārqos </placeName>
+                    <country ref="wd:Q115">Ethiopia</country>
+                    <settlement ref="LOC4508Manbar"/>
```
Comment (eu-genia) [inline new/INS0984MLQM.xml:51]: this is not the settlement, this is the same monastery, as LOC, this should go to a relation (many of institutions also have LOC records, there was a long discussion whether LOC records should be removed but it was decided that they stay and are linked at that point) https://github.com/BetaMasaheft/Documentation/issues/1032

File: new/INS0985MSDM.xml [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="DR" when="2022-09-22">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="privateHouse" subtype="institution">
+                    <placeName>Private collection of Mamhǝr Salomon (Dabra Mārqos)</placeName>
```
Comment (eu-genia) [inline new/INS0985MSDM.xml:48]: mamḫǝr?

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/INS0983MHDL.xml (hunk 5 above)
Comment (thea-m) [reply new/INS0983MHDL.xml:48]: I followed the spelling I was given...

**Comment by thea-m** [comment]
Thank you! You are of course right, but my time and energy for non-core projects are limited, and for this I will prefer to channel them into the training necessary for the manuscript descriptions, and keep the other involved records minimal.

File: new/INS0984MLQM.xml [hunk 9]
```diff
@@ -48,9 +48,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <placeName xml:lang="gez" xml:id="n1">መንበረ፡ ልዑል፡ ቅዱስ፡ ማርቆስ፡</placeName>
                     <placeName xml:lang="gez" corresp="#n1" type="normalized">Manbara Lǝʿul Qəddus Mārqos </placeName>
                     <country ref="wd:Q115">Ethiopia</country>
-                    <settlement ref="LOC4508Manbar"/> 
                 </place>
             </listPlace>
+            <div type="bibliography">
```
Comment (eu-genia) [inline new/INS0984MLQM.xml:53]: thank you! we still sould need region and settlement, or the visualization would be with empty spaces and commas 
the church is in Addis Ababa (Sidist Kilo I believe)

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/INS0983MHDL.xml (hunk 5 above)
Comment (eu-genia) [reply new/INS0983MHDL.xml:48]: By whom? The file names we get the names from have no diacritics by definition...

**Comment by eu-genia** [comment]
> Thank you! You are of course right, but my time and energy for non-core projects are limited, and for this I will prefer to channel them into the training necessary for the manuscript descriptions, and keep the other involved records minimal.

Sure, I understand, I was only thinking of the way the names of institutions and especially the manuscript shelfmarks will be rendered.

File: new/INS0984MLQM.xml (hunk 9 above)
Comment (thea-m) [reply new/INS0984MLQM.xml:54]: oh, sorry, that's true! I tried adding more information.

File: new/INS0984MLQM.xml [hunk 10]
```diff
@@ -48,6 +48,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <placeName xml:lang="gez" xml:id="n1">መንበረ፡ ልዑል፡ ቅዱስ፡ ማርቆስ፡</placeName>
                     <placeName xml:lang="gez" corresp="#n1" type="normalized">Manbara Lǝʿul Qəddus Mārqos </placeName>
                     <country ref="wd:Q115">Ethiopia</country>
+                    <settlement ref="wd:Q3624">ʾAddis Ababā</settlement>
```
Comment (eu-genia) [inline new/INS0984MLQM.xml:51]: sorry for nitpicking, did we not say we give internal LOC refs and not wd for places, to be able to see attestations etc.?

**Review by eu-genia — APPROVED** [review]

**Review by eu-genia — APPROVED** [review]

**Pull request merged**